Sources/Prometheus/RELEASE.md

```diff
@@ -0,0 +1,50 @@
+# Releasing 
+
+## Release schedule
+
+Releases of this library are performed on a when-needed basis. 
+
+Generally one of the maintainers listed in MAINTAINERS.md will perform a release.
+
+If you would like to request a release since a change or fix has been waiting for a release for some time,
+you may open up an issue on github, requesting a release.
+
+## Versioning 
+
+As usual with Swift packages, we use [Semantic Versioning](https://semver.org/).
+
+## Make a release
+
+Development and releases are made from the `main` branch directly, therefore the branch should remain in releasable state at any time.
+
+Making a release is simple and boils down to making a git tag and pushing it to the upstream repository.
+
+Tags should follow the simple `MAJOR.MINOR.PATCH` format, without any prefix (e.g. do not include the `v` prefix).
```

Comment:
I’m curious about the reason behind this decision in the past, as per GitHub common practices, it’s recommended to use the v prefix.

Reply:
It's a SwiftPM thing, it seems it can properly strip the `v` on desktop but seems to have had bugs where not all envs would handle that 

> This incorrect behavior is specific to Swift Playgrounds for iPad. SPM in the command line, SPM embedded in Xcode, and other package managers such as CocoaPods deal nicely with "v..." tags.

https://forums.swift.org/t/support-for-v-prefixed-version-tags-in-swiftpm-swift-playgrounds-for-ipad/54845

Reply:
Ty for sharing this!